crates/networking/rpc/clients/eth/mod.rs

```diff
@@ -1198,6 +1193,9 @@ impl EthClient {
         &self,
         maybe_gas_fee: Option<u64>,
     ) -> Result<u64, EthClientError> {
+        if let Some(gas_fee) = maybe_gas_fee {
+            return Ok(gas_fee);
+        }
         Ok(maybe_gas_fee.unwrap_or(
```

Comment:
here maybe_gas_fee is None for certain so you can skip this unwrap and call get_gas_price directly

Reply:
Done

Reply:
+1